modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLayout from '@clayui/layout';
+import PropTypes from 'prop-types';
+import React, {useCallback, useState} from 'react';
+
+import {useChartState} from '../state/chartState';
+
+export default function Translation({
+	defaultLanguage,
+	defaultTimeSpanOption,
+	onSelectedLanguageClick,
+	publishDate,
+	viewURLs,
+}) {
+	const [active, setActive] = useState(false);
+
+	const {state: chartState} = useChartState({
+		defaultTimeSpanOption,
+		publishDate,
+	});
+
+	const {timeSpanOffset, timeSpanOption} = useCallback(() => {
```

Comment:
Yes! The idea is to get the `timeSpanKey` (before `timeSpanOption` 🤷‍♀️ ) and the `timeSpanOffset` to keep that configuration when the user changes the language to view the performance of the content in another language. I'll double-check this part because this change a little bit last week. Thanks